src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs

```diff
@@ -3166,7 +3166,7 @@ private void CreateContainerProcessInternal()
                     // Windows Server container (i.e., RuntimeId is empty) uses named pipe transport for now.
                     //
                     cmd = string.Format(System.Globalization.CultureInfo.InvariantCulture,
-                        @"{{""CommandLine"": ""powershell.exe {0} -NoLogo {1}"",""RestrictedToken"": {2}}}",
+                        @"{{""CommandLine"": ""pwsh.exe {0} -NoLogo {1}"",""RestrictedToken"": {2}}}",
```

Comment:
Same issue here in that we rely on a "built-in" pwsh.exe command that is discoverable.  But we have the same issue with ssh.exe used for SSH based remoting.

Reply:
Actually as I think about it, we don't want to change this here.  This is a command run on a target VM and relies on the built-in Windows PowerShell.  pwsh.exe will not be available on a new VM.

Reply:
So this command is run within the VM expecting Windows PowerShell?

Reply:
Yes

Reply:
Ok, will change back.